src/activities/CategoriesEntity.js

```diff
@@ -165,13 +165,17 @@ export class CategoriesEntity extends Entity {
 		return categoryId !== initialId;
 	}
 
-	async save(category, shouldDeletePreviousCategory = false) {
+	async save(category, shouldDeletePreviousCategory = null) {
```

Comment:
This prevents FACE Discussions from generating `select` & `deselect` actions for saving.